packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -275,12 +275,15 @@ Media.updateTransceiver = ({meetingId, remoteQualityLevel}, peerConnection, tran
 
 /**
  * generates share streams
- * @param {object} options parameter
- * @param {Object} options.sendAudio sendAudio: {Boolean} sendAudio constraints
- * @param {Object} options.sendShare sendShare: {Boolean} sendShare constraints
- * @param {Object} options.sharePreferences sharePreferences: {Object} Share constraints and share constraints
+ * @param {Object} options parameter
+ * @param {Boolean} options.sendAudio send audio from the display share
+ * @param {Boolean} options.sendShare send video from the display share
+ * @param {Object} options.sharePreferences
+ * @param {MediaTrackConstraints} options.sharePreferences.shareConstraints constraints to apply to video
+ *   @see {@link https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints}
+ * @param {Boolean} options.sharePreferences.highFrameRate if shareConstraints isn't provided, set default values based off of this boolean
  * @param {Object} config SDK Configuration for meetings plugin
- * @returns {Object} {streams}
+ * @returns {Promise.<MediaStream>}
  */
 Media.getDisplayMedia = (options, config) => {
```

Comment:
@adamweeks integration tests are failing due to config not having defaults.
In the scenario where **only** `highFrameRate` is passed this function will fail due to config being undefined.
Now setting defaults would be simple, but the problem on top of that would be that both ternary values rely on a `config`. 
the reason this didn't get all before was because `getMediaStreams` was grabbing a bunch of `sharePreferences` props before calling this function (somewhat indirectly).

Also `const shareConstraints = options.sharePreferences && options.sharePreferences.shareConstraints || {` makes no sense and is probably too "busy" to understand on 1st-3rd glance